### PR TITLE
*: add some comments to exported function, part of enable revive.exported

### PR DESCRIPTION
**Pull request by okJiang**
### What problem does this PR solve?

Issue Number: Ref https://github.com/tikv/pd/issues/8458, https://github.com/tikv/pd/pull/8440#discussion_r1696647879

### What is changed and how does it work?

```commit-message
```

### Check List

Tests

- Unit test
- Integration test
- Manual test (add detailed scripts or steps below)
- No code

Code changes

- Has the configuration change
- Has HTTP API interfaces changed (Don't forget to [add the declarative for the new API](https://github.com/tikv/pd/blob/master/docs/development.md#updating-api-documentation))
- Has persistent data change

Side effects

- Possible performance regression
- Increased code complexity
- Breaking backward compatibility

Related changes

- PR to update [`pingcap/docs`](https://github.com/pingcap/docs)/[`pingcap/docs-cn`](https://github.com/pingcap/docs-cn):
- PR to update [`pingcap/tiup`](https://github.com/pingcap/tiup):
- Need to cherry-pick to the release branch

### Release note

```release-note
None.
```

### CODE REVIEW

File: .golangci.yml [hunk 1]
```diff
@@ -214,3 +214,6 @@ issues:
     - path: (pd-analysis|pd-api-bench|pd-backup|pd-ctl|pd-heartbeat-bench|pd-recover|pd-simulator|pd-tso-bench|pd-ut|regions-dump|stores-dump)
       linters:
         - errcheck
+  include:
+    # remove the comment after the path is ready
```
Comment (HuSharp) [inline .golangci.yml:218]: what does this `path` mean?

Comment (okJiang) [reply .golangci.yml:218]: After adding comments to all exported name/function, these two lines of comments will be removed.

Comment (okJiang) [reply .golangci.yml:218]: This PR is just a partial part.

**Review by HuSharp — APPROVED** [review]

File: .golangci.yml (hunk 1 above)
Comment (HuSharp) [reply .golangci.yml:218]: got it

Comment (HuSharp) [reply .golangci.yml:218]: maybe can linked with #8440 in pr description :-)

Comment (okJiang) [reply .golangci.yml:218]: No problem. BTW, it has been linked to the issue

**Review by rleungx — APPROVED** [review]

**Pull request merged**